lib/src/actions/actions.dart

```diff
@@ -1409,6 +1409,18 @@ abstract class DeleteAllSelected
   String short_description() => "remove all selected items";
 }
 
+// Selects all selectable items in the current helix group.
```

Comment:
Instead of creating a new Action, I think it would be cleaner to add a Boolean field `current_helix_group_only` to the existing action `SelectAllSelectable`. Then the reducer `select_all_selectables_reducer` would essentially just contain the code currently in `select_all_selectables_helper` but would check that field instead of the extra parameter `only_in_helix_group`.